docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -91,6 +91,67 @@ func (ds RatioBasedSampler) Description() string {
 }
 ```
 
+### Filtering
+
+In some cases it may be desirable to reduce the number of metrics produced by the `spanmetrics` processor. This can be accomplished by configuring the processor to use a filter. An `include` filter is used to match criteria that must be present in the span in order to be included. Following the include filter, an `exclude` filter may be used to reject portions of what was previously included by the filter policy.
+
+Currently supported is filtering by resource and span attributes with the following value types.
+
+- string
+- int
+- double
+
+Additionally, the following intrinsic span attributes may be filtered upon.
+
+- name
+- status (code)
+- kind
```

Comment:
Are these values that would appear in the code snippet? Should they be formatted with `? In other words: 
```suggestion
- `name`
- `status` (code)
- `kind`
```